packages/cli/src/build/config.ts

```diff
@@ -181,24 +181,9 @@ export function getRollupConfigs(pkg: Package, aliases: Aliases) {
       });
     });
 
-  let hasBrowserCondition = false;
-  let hasWorkerCondition = false;
-  if (pkg.project.experimentalFlags.exports) {
-    if (typeof pkg.json.exports == "object") {
-      hasBrowserCondition = Object.values(pkg.json.exports!).some(
-        (condition) =>
-          typeof condition === "object" && condition.browser !== undefined
-      );
-      hasWorkerCondition = Object.values(pkg.json.exports!).some(
-        (condition) =>
-          typeof condition === "object" && condition.worker !== undefined
-      );
-    }
-  }
-
   let hasBrowserField = pkg.entrypoints[0].json.browser !== undefined;
 
-  if (hasBrowserField || hasBrowserCondition) {
```

Comment:
do we mandate `envConditions` to include `'browser'` when there is `package.json#browser` already?

Reply:
yes